ios/RNFirebase/notifications/RNFirebaseNotifications.m

```diff
@@ -100,10 +102,17 @@ - (void)didReceiveLocalNotification:(nonnull UILocalNotification *)localNotifica
 
 RCT_EXPORT_METHOD(complete:(NSString*)handlerKey fetchResult:(UIBackgroundFetchResult)fetchResult) {
     if (handlerKey != nil) {
-        void (^completionHandler)(UIBackgroundFetchResult) = completionHandlers[handlerKey];
-        if(completionHandler != nil) {
-            completionHandlers[handlerKey] = nil;
-            completionHandler(fetchResult);
+        void (^fetchCompletionHandler)(UIBackgroundFetchResult) = fetchCompletionHandlers[handlerKey];
+        if (fetchCompletionHandler != nil) {
+            fetchCompletionHandlers[handlerKey] = nil;
+            fetchCompletionHandler(fetchResult);
+        }
+        else {
+            void(^completionHandler)(void) = completionHandlers[handlerKey];
```

Comment:
Any reason for removing the previous behaviour of using `UIBackgroundFetchResult`? Is this not going to break the previous behaviour? I might be getting a little confused on the changes here so apologies in advance 🙈

Reply:
The NSUserNotificationCenter completion handler doesn't take any arguments in its signature so you just need to call it.